### PR TITLE
Refactor import of store models

### PR DESCRIPTION
**Pull request by seallard**
## Description
We have previously refactored small parts of the codebase to use a more readable form of imports of the store models. Since this clutters the diffs of our other pull requests, I refactored the entire codebase to use the model specific imports from the store.

So all instances of:

```python
from cg.store import models
from cg.store import Store, models
```

Have been refactored to
```python
from cg.store.models import ...
```

Black applied some formatting as well, but nothing more than the imports should have changed. It is a boring PR - but I think it saves us some time and energy in the long run.

## Review
- [ ] "Merge and deploy" approved by

### This [version](https://semver.org/) is a
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

File: tests/meta/rsync/test_rsync.py [hunk 1]
```diff
@@ -9,14 +9,14 @@
 from cgmodels.cg.constants import Pipeline
 from cg.exc import CgError
 from cg.meta.rsync import RsyncAPI
-from cg.models.cg_config import CGConfig
```
Comment (diitaz93) [inline tests/meta/rsync/test_rsync.py:12]: I would double check this deletion as it is another module

**Review by diitaz93 — CHANGES_REQUESTED** [review]
Great initiative! just double check my commented line

File: tests/meta/rsync/test_rsync.py (hunk 1 above)
Comment (seallard) [reply tests/meta/rsync/test_rsync.py:12]: It was unused - so no worries :)

**Review by Vince-janv — APPROVED** [review]
Great initiative! 💡

**Review by diitaz93 — APPROVED** [review]

**Comment by seallard** [comment]
Deployed to stage and prod.

**Pull request merged**